index.md

```diff
@@ -3,130 +3,181 @@ title: Cockpit Project
 layout: essential
 class: front-page
 ---
-{% include page_header.html %}
+{% capture newline %}
+{% endcapture %}
+
+{% capture screenshots %}
+login-opt.png: Cockpit's log in prompt (on Fedora 34)
+debian-in-windows-edge.jpg: Cockpit works where you are (Pictured: Connecting to Debian server from Microsoft Edge on Windows 10)
+journal.png: View, filter, and search system logs
+accounts.png: Edit accounts
+firewall-rhel.png: Edit the firewall with ease (Pictured: Cockpit Web Console on Red Hat Enterprise Linux, connected from Fedora 34 Workstation)
```

Comment:
This screenshot has a *lot* of desktop background, was that intended? It makes the fonts quite a bit blurry.

Reply:
Yeah, I threw in a couple full screenshots and some cropped to browser UI to show context. I wanted to switch it up and show that it runs in Windows and Linux (and to show off that Linux actually has desktops UIs too; it's not just for servers).

They're _heavily optimized_ for file size. And they're not blurry at all, unless the browser scales it down. It's possible to middle click, control-click, or right click (and view link) to open in tabs and see the file 1:1 in the browser as well.

Reply:
We could also swap out the page shown if highlighting the firewall makes more sense.